### DISCUSSION TITLE
React.js - Pedido de indicação de curso

### DISCUSSION THREAD

**Discussion by caiqalmeida** [frontendbr/forum#1529] [Discussão]
Olá pessoal !

Esse ano iniciei meus estudos em javaScript, e no momento, estou finalizando o curso de javaScript do @fdaciuk , que é excelente ! Após isso, quero fazer 3 projetos médios/grandes e após isso, prosseguir para React que é o que o mercado da minha região (Sorocaba/SP) mais tem oportunidade (estou em busca de um estágio em front-end).

Com isso, podem me dar indicações de curso ? Meu objetivo é o de React do próprio Daciuk, mas infelizmente no momento não estou podendo arcar, vou deixar mais para o fim do ano até ter alguma promoção boa  e eu tiver mais tranquilo financeiramente. Até lá, tem algum curso da udemy pelos famosos 20 ~ 30 reais que valham a pena ?

Agradecido por qualquer comentário :)

**Comment by weider86-zz** [comment]
Fala Caíque, blz?

Estou acompanhando o ReactNinja do @fdaciuk <https://github.com/fdaciuk>e é
realmente muito bom!

Pra começar com algo mais de leve e ainda por cima na faixa, tenta os
Starter da RocketSeat:
https://rocketseat.com.br/starter

Sucesso na sua caminhada!

Em qui, 5 de set de 2019 às 12:52, Caique Almeida <notifications@github.com>
escreveu:

> Olá pessoal !
>
> Esse ano iniciei meus estudos em javaScript, e no momento, estou
> finalizando o curso de javaScript do @fdaciuk <https://github.com/fdaciuk>
> , que é excelente ! Após isso, quero fazer 3 projetos médios/grandes e após
> isso, prosseguir para React que é o que o mercado da minha região
> (Sorocaba/SP) mais tem oportunidade (estou em busca de um estágio em
> front-end).
>
> Com isso, podem me dar indicações de curso ? Meu objetivo é o de React do
> próprio Daciuk, mas infelizmente no momento não estou podendo arcar, vou
> deixar mais para o fim do ano até ter alguma promoção boa e eu tiver mais
> tranquilo financeiramente. Até lá, tem algum curso da udemy pelos famosos
> 20 ~ 30 reais que valham a pena ?
>
> Agradecido por qualquer comentário :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/1529?email_source=notifications&email_token=ACAAGX2PL3MNJGGFRGGT2CTQIETM7A5CNFSM4IT7SIC2YY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HJSRMLQ>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACAAGX7PACG26DTS57LFDS3QIETM7ANCNFSM4IT7SICQ>
> .
>


-- 
*Abs.*

*Weider Lima*
*Web Frontend Developer *
*(11) 9.8569.3927 - Whatsapp*
https://br.linkedin.com/in/weiderlima

**Comment by yuritoledo** [comment]
Não sei seu nível de inglês, mas uma ótima fonte de conhecimento é o [freecodecamp](https://freecodecamp.org) . É free e dá certificado!

Também é bacana para dar uma enriquecida no vocabulário, com os termos e a leitura em si.

**Comment by ViniciusGularte** [comment]
Além do que o pessoal citou, recomendo a [docs](https://pt-br.reactjs.org/docs/getting-started.html) do react, sempre dou uma consultada e me ajudou bastante no começo (ainda segue ajudando).

**Comment by caiqalmeida** [comment]
Manos,

Vou seguir as dicas de vocês e seguir da seguinte maneira : 
1. Documentação do React para os passos iniciais
2. O Daciuk fez uma promo do curso dele e vou dar uma apertada pra conseguir comprar e faze-lo

O freecode camp seria minha alternativa se eu não fosse comprar o curso. Fiz HTML, CSS e JavaScript basics lá e é excelente ! 

Muito obrigado pela atenção de todos ! Espero que sirva para algum próximo interessado que for procurar este assunto.

Abraço

**Comment by fabiomirandaa** [comment]
> Fala Caíque, blz? Estou acompanhando o ReactNinja do @fdaciuk <https://github.com/fdaciuk>e é realmente muito bom! Pra começar com algo mais de leve e ainda por cima na faixa, tenta os Starter da RocketSeat: https://rocketseat.com.br/starter Sucesso na sua caminhada! Em qui, 5 de set de 2019 às 12:52, Caique Almeida <notifications@github.com> escreveu:
> […](#)
> Olá pessoal ! Esse ano iniciei meus estudos em javaScript, e no momento, estou finalizando o curso de javaScript do @fdaciuk <https://github.com/fdaciuk> , que é excelente ! Após isso, quero fazer 3 projetos médios/grandes e após isso, prosseguir para React que é o que o mercado da minha região (Sorocaba/SP) mais tem oportunidade (estou em busca de um estágio em front-end). Com isso, podem me dar indicações de curso ? Meu objetivo é o de React do próprio Daciuk, mas infelizmente no momento não estou podendo arcar, vou deixar mais para o fim do ano até ter alguma promoção boa e eu tiver mais tranquilo financeiramente. Até lá, tem algum curso da udemy pelos famosos 20 ~ 30 reais que valham a pena ? Agradecido por qualquer comentário :) — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#1529?email_source=notifications&email_token=ACAAGX2PL3MNJGGFRGGT2CTQIETM7A5CNFSM4IT7SIC2YY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HJSRMLQ>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACAAGX7PACG26DTS57LFDS3QIETM7ANCNFSM4IT7SICQ> .
> -- *Abs.* *Weider Lima* *Web Frontend Developer * *(11) 9.8569.3927 - Whatsapp* https://br.linkedin.com/in/weiderlima

@weider86  Esse curso do Daciuk está atualizado?

**Comment by wilsonneto-swe** [comment]
@caiqalmeida se ainda estiver em busca, o da rocketseat ta com as inscrições abertas, mas está a R$ 1980,00... :/

BootCamp Rocketseat: https://rocketseat.com.br/gostack-9/inscricao (só até o final dessa semana)

Eu comprei ele, e acho que paguei até mais caro e valeu muito a pena... porém caro rs...

Mas o primeiro curso que fiz mesmo foi o do dev pleno que achei tão bom quanto e foi meu início... ai assim que comecei a ganhar melhor me tornando fullstack (eu era backend em .Net Core, hoje fullstack com .Net Core, ReactJS e um pouco de node).

O Curso do Dev Pleno é R$ 697, achei o custo beneficio bem legal aqui: [Dev Pleno](https://go.hotmart.com/T14343972J)

E também a documentação, claro, mas eu gosto de fazer cursos para codar projetos práticos, e quando é qualidade certa eu não ligo em comprar, já que é investimento.

Abraços!

**Comment by caiqalmeida** [comment]
@wilsonneto-dev olá, me desculpa a demora amigo ! 

Caraca, sério que vale a pena assim ? O cara deve ser muito bom mesmo ahahaaha 

Concordo com você, não me importo em pagar por ser um investimento que com certeza vai dar retorno né.

Mas infelizmente não tenho condição de fazer essa aporte agora :/ 

Ainda não iniciei os estudos em React, pq comecei a fazer algumas coisas, mas estava barrando em algumas coisa de javascript ainda. Achei melhor reforçar a base antes de avançar. 

Estou tentando fazer projetos práticos. Se alguém tiver alguma dica do que é interessante criar com js
